signer/contentsignature/contentsignature_test.go

```diff
@@ -218,3 +219,21 @@ func TestNoShortData(t *testing.T) {
 		t.Fatalf("expected to fail with input data too short but failed with: %v", err)
 	}
 }
+
+// Reproduce nil error
```

Comment:
This comment doesn't explain what's going on. I think we could leave it out.

Reply:
Resolved [7b19fee](https://github.com/mozilla-services/autograph/pull/1033/commits/7b19fee2a2e3daac50d64af473f12aa4ea1f72c8)